.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve our plugin
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Bug description**
+A clear and concise description of what the bug is.
+
+**Steps to reproduce**
```

Comment:
Hmm I'd mention that attaching the logs would be useful - similar stuff as in README.md/DEVELOPMENT.md, maybe a bit abridged